glob-pattern.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+const glob = require('globby');
+const junk = require('junk');
+const {join, basename, isAbsolute} = require('path');
+const {existsSync, lstatSync} = require('fs');
```

Comment:
This is not a good use for destructuring. Seeing `join()` in the middle of the code means nothing. Context is important. So I would prefer not using restructuring here. Same in other places.

Reply:
Good point. I've switched to default import